modules/accredited_representative_portal/app/services/accredited_representative_portal/power_of_attorney_request_service/params_schema.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  module PowerOfAttorneyRequestService
+    module ParamsSchema
+      # Load extensions needed for schema validation
+      Dry::Schema.load_extensions(:json_schema)
+      Dry::Schema.load_extensions(:hints)
+
+      ALLOWED_SORT_FIELDS = %w[created_at updated_at].freeze
```

Comment:
Is this a straggler that can be deleted?

Reply:
yup! 👍